cmd/server.go

```diff
@@ -34,7 +34,7 @@ func init() {
 	rootCmd.AddCommand(serverCmd)
 
 	serverCmd.Flags().StringVar(&serverPdAddr, "pd", "http://127.0.0.1:2379", "PD address, separated by comma")
-	serverCmd.Flags().StringVar(&statusAddr, "status-addr", "0.0.0.0:8300", "Bind address for http status server")
+	serverCmd.Flags().StringVar(&statusAddr, "status-addr", "127.0.0.1:8300", "Bind address for http status server")
```

Comment:
DBA suggests setting the default value for `status host` to `0.0.0.0`, the same with TiDB

Reply:
0.0.0.0 in capture info is meaningless. 127.0.0.1 makes user assign a meaningful address.